### PR TITLE
feat: add @asirialwis as a contributor

### LINKED ISSUE #322: Feature: Add @asirialwis as a contributor
### Description

As part of the Intro to Open Source course, we are encouraged to contribute to this repository.

### Suggested solution

Add myself as a contributor.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Getting Started Instructions

- [X] I agree to follow this project's Getting Started instructions

### PR DESCRIPTION
**Pull request by asirialwis**
## Description

This PR add  @asirialwis as a contributor.

## What type of PR is this? (check all applicable)

- 🤝 Add a contributor
- [ ] 📝 Documentation Update

### Related Issues

Closes #322

### I've read through the [Getting Started](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=getting-started) section.

- ✅ Yes
- [ ] ❌ Not yet

### Have you run `npm run contributors:generate` to generate your profile and the badge on the README?

- ✅ Yes
- [ ] ❌ No

## Added to documentation?

-  📜 README.md
- [ ] 🙅 no documentation needed

## Screenshot (Required for PR Review)
Fosspng

### CODE REVIEW

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @asirialwis,
Thank you for your contribution! ✨ 

There are a few things here:

- Please [create an issue as mention in our course](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=getting-started) to accompany this PR as it's the best practice in open source.
- You have a [failed check](https://github.com/open-sauced/guestbook/pull/320#pullrequestreview-2063867975) that you need to address.
- I see that you haven't completed your PR template and you've deleted a section here. That said, please put back the section and fill it in (you can look at previous PRs to see what you've removed) and complete your PR template so we can understand the changes you made and the background of this PR.

You can read this [blog post](https://dev.to/opensauced/streamline-your-contributions-mastering-issue-forms-and-pr-templates-36j5) to learn the importance of templates and how to fill in a PR template.

We're only reviewing this PR once we get all the information needed.
Please let us know once you've completed the template or if you have any question.

Thank you. 🙂

**Comment by adiati98** [comment]
Hey @asirialwis,

We have merged a PR before yours. Please **[resolve the conflicts](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=merge-conflicts-in-the-guestbook-repository)** before we can review your PR.

Do let us know if you need any help. Thanks. 😊

**Comment by asirialwis** [comment]
I have solved the merge conflicts.

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey there,

I highly recommend you read and follow the [instructions to resolve merge conflicts in this repo](https://intro.opensauced.pizza/#/intro-to-oss/how-to-contribute-to-open-source?id=merge-conflicts-in-the-guestbook-repository).

From what I see here, you haven't run the command to generate your profile on README. And I think once you run it, it will throw an error. But don't get discouraged and just follow the instructions. 🙂

Let me know if you have any questions or need any help. ✨️

File: .all-contributorsrc [hunk 1]
```diff
@@ -2076,6 +2076,26 @@
...
+        "design",
+        "doc",
+        "example",
+        "ideas",
+        "projectManagement",
+        "research",
+        "review",
+        "talk",
+        "translation"
+
```
Comment (StackOverflowIsBetterThanAnyAI) [inline .all-contributorsrc:2098]: Please close the array and the object accordingly.

**Review by StackOverflowIsBetterThanAnyAI — CHANGES_REQUESTED** [review]

**Review by adiati98 — APPROVED** [review]
Hi @asirialwis,
We've helped you resolve the merge conflicts in your PR. ✨ 

For your future reference, please read and follow any instructions carefully. You can see the [commit history](https://github.com/open-sauced/guestbook/pull/320/commits) in this PR to understand the changes that I made to resolve the conflicts.

If you haven't, you can join our community [Discord](https://discord.com/invite/U2peSNf23P). 😊
Also, check out our [pizza-verse](https://github.com/open-sauced/pizza-verse) repo if you want to contribute further! 🍕

pizza GIF

**Pull request merged**